src/psyclone/tests/psyir/backend/fortran_test.py

```diff
@@ -2153,3 +2153,40 @@ def test_fw_operand_clause(fortran_writer):
     ref_a = Reference(symbol_a)
     op_clause.addchild(ref_a)
     assert "depend(inout: a)" in fortran_writer(op_clause)
+
+
+def test_fw_common_blocks(fortran_reader, fortran_writer):
```

Comment:
Not yours but, since I noticed it, please could you fix the pylint warning about unused `monkeypatch` at L127?

Reply:
Done